buildSrc/src/test/java/org/elasticsearch/gradle/precommit/DependencyLicensesTaskTests.java

```diff
@@ -57,16 +56,19 @@ public void givenProjectWithLicensesDirButNoDependenciesThenShouldThrowException
     }
 
     @Test
-    public void givenProjectWithoutLicensesDirButWithDependenciesThenShouldThrowException() throws Exception {
+    public void givenProjectWithoutLicensesDirButWithDependenciesThenShouldThrowException()
+            throws Exception {
         expectedException.expect(GradleException.class);
-        expectedException.expectMessage(containsString("does not exist, but there are dependencies"));
+        expectedException.expectMessage(
+                containsString("does not exist, but there are dependencies"));
 
         project.getDependencies().add("compile", dependency);
         task.get().checkDependencies();
     }
 
     @Test
-    public void givenProjectWithoutLicensesDirNorDependenciesThenShouldReturnSilently() throws Exception {
+    public void givenProjectWithoutLicensesDirNorDependenciesThenShouldReturnSilently()
```

Comment:
Do we have a different line length setting in the formatter than in checkstyle ? 
Not sure why this line was changed

Reply:
GJF has a line length of 100, as opposed to 140 for Checkstyle.

Reply:
That's an item that we were considering forking `google-java-format` for, not because we necessarily think that 140 is the right line length (I don't), but because we wanted to avoid a *massive* reformatting of the codebase.